tests/Compilation/positive/test045.juvix

```diff
@@ -0,0 +1,14 @@
+-- implicit builtin bool
```

Comment:
can we add a brief comment explaining why this test requires builtin bool?

Reply:
Done